### PR TITLE
feat: added placeholder to contributors list on project page gf-377

### PR DESCRIPTION
**Pull request by Vitaliistf**
Added a placeholder, which looks like this:

Screenshot 2024-09-18 at 13 36 09

P.S. had to split conditions in order to get rid of linter problems. Linter does not allow to nest ternary expression without parentheses, but formatter wants that expression without them. If there is a better solution, please suggest it.

### CODE REVIEW

File: apps/frontend/src/pages/project/libs/components/contributors-list/contributors-list.tsx [hunk 1]
```diff
@@ -13,12 +14,13 @@ const ContributorsList = ({
...
+	const hasContributors = contributors.length > EMPTY_LENGTH;
+
 	return (
 		<div className={styles["container"]}>
 			<h2 className={styles["title"]}>Contributors</h2>
-			{isLoading ? (
-				<Loader />
-			) : (
+			{isLoading && <Loader />}
+			{!isLoading && hasContributors && (
```
Comment (perekotypole) [inline apps/frontend/src/pages/project/libs/components/contributors-list/contributors-list.tsx:23]: How about returning <Loader/> before, and don't use so many conditions for each block?

Comment (Vitaliistf) [reply apps/frontend/src/pages/project/libs/components/contributors-list/contributors-list.tsx:23]: Looks like a nice solution, let's see what will Liza say about it. The only thing is that we won't have this layout of a component at all (container and title). I just tried to do it as we've done before in table and projects page.

Comment (liza-veis) [reply apps/frontend/src/pages/project/libs/components/contributors-list/contributors-list.tsx:23]: I think in this case we will miss the title, so I would leave it

**Comment by liza-veis** [comment]
I think it would be better to add some space between title and placeholder

**Comment by Vitaliistf** [comment]
> I think it would be better to add some space between title and placeholder

Added more space.

File: apps/frontend/src/pages/project/libs/components/contributors-list/contributors-list.tsx (hunk 1 above)
Comment (liza-veis) [inline apps/frontend/src/pages/project/libs/components/contributors-list/contributors-list.tsx:23]: let's move these conditions to separate constants - isListShown and isEmptyPlaceholderShown
```suggestion
			{isListShown && (
```

**Review by liza-veis — CHANGES_REQUESTED** [review]

**Review by liza-veis — APPROVED** [review]

**Pull request merged**